test/checkoutput/README.md

```diff
@@ -0,0 +1,54 @@
+# checkoutput tool
+
+The checkoutput tool parses a text or markdown that has a command and the expected output sequence, runs the command and checks the output of the command against the expected output. 
+
+**Usage:**
+```
+<tool> -parsefile <path/to/file> -verbose
```

Comment:
Instead of `-verbose`, perhaps enable logging options?

I think I prefer `<tool> <directory> <directory> ...` as invocation. Then the tool can check the binary defined in each directory based on the files defined there.

We can start with test cases embedded in .md files. Supporting comments in .go files could be added later, if we find a need for it.

Reply:
I don't understand the alternative for -verbose. What does it mean to enable logging options ? Can you please elaborate.

Reply:
Enable `-v` and then in the tool, use `klog.V(...).InfoS` to print information at various level of detail.

This can be done as in https://github.com/kubernetes/kubernetes/tree/master/staging/src/k8s.io/component-base/logs/example